app/assets/stylesheets/application.scss

```diff
@@ -26,6 +26,10 @@ $component-active-bg: transparent;
 
 $font-family-sans-serif: 'Work Sans', sans-serif;
 
+#alert-message{
+  margin: 0px
+}
```

Comment:
Since bootstrap is being entirely removed, you may want to create a new style for the errors